branchpro/models.py

```diff
@@ -80,11 +80,12 @@ class BranchProModel(ForwardModel):
     Methods
     -------
     simulate: return model output for specified parameters and times.
-    get_serial_intervals: returns serial intevals for the model.
-    update_serial_intevals: updates serial intevals for the model.
+    get_r_profile: returns R_t profile for the model.
+    get_serial_intervals: returns serial intervals for the model.
+    update_serial_intervals: updates serial intervals for the model.
     set_r_profile: creates a new R_t profile for the model.
 
-    *Always apply method set_r_profile before simulation for a change of R_t
+    *Always apply method set_r_profile before simulate for a change of R_t
```

Comment:
```suggestion
*Always apply method `set_r_profile` before simulating for a change of R_t
```

Reply:
Maybe I can use the quotation for simulate? I mean, simulate is a method for the class. What do you think?

Reply:
Ah I didn't catch that you were referring to the method. I think you can do this:

```suggestion
    *Always apply method `set_r_profile` before calling :meth:`BranchProModel.simulate` for a change of R_t
```

The `:meth:` directive will render it to a link to the simulate method in the docs